lib/is-element.test.js

```diff
@@ -2,24 +2,24 @@
 
 var assert = require("@sinonjs/referee").assert;
 
-// in order for `is-element.js` to correctly detect the DOM being set up
-// in the previous line, we need to reload it, as other tests will have
-// loaded `is-element.js`, most likely without a DOM in place
-function requireWithDOM(file) {
-    require("jsdom-global")();
-
-    if (require.resolve) {
-        delete require.cache[require.resolve(file)];
-    }
-
-    return require("./is-element");
-}
-
 describe("isElement", function() {
-    var isElement;
+    var isElement = require("./is-element");
```

Comment:
I don't think the require statement here does anything we need?

Reply:
That's right. I will remove it, thanks 👌